challenge_two/src/utils/getPoolAddress.ts

```diff
@@ -0,0 +1,47 @@
+import { LRUCache } from "typescript-lru-cache";
+import { computePoolAddress, FeeAmount } from "@uniswap/v3-sdk";
+import { Token } from "@uniswap/sdk-core";
+import { Contract } from "ethers";
```

Comment:
how does this line work? there's no "ethers" library in the code.

I would suggest importing ethers from "forta-agent", and then calling ethers.Contract